pkg/nsx/services/vpc/vpc.go

```diff
@@ -45,13 +45,23 @@ var (
 	enableAviAllowRule = false
 )
 
+type VPCNetworkInfoStore struct {
```

Comment:
Directly embed sync.Mutex and s.Lock() instead of s.VPCNetworkConifgStore.m.Lock(), this way is better?